src/puppetlabs/puppetdb/http/resources.clj

```diff
@@ -16,7 +16,8 @@
                   version
                   (params "query")
                   paging-options
-                  (:scf-read-db globals)))
+                  (:scf-read-db globals)
+                  (:url-prefix globals)))
                http-q/restrict-query-to-active-nodes)}))
```

Comment:
hehehe, well it won't during a follow as I've said, /catalogs/nodes/resources doesn't have an active node restrict. So for this endpoint more than anything, its dangerous to change this default since exported resources would expect this to be restricted. That is, it would need a change to the terminus and potentially we'd need to let people know we're breaking this for puppetdbquery queries and such that exist in the wild. I think ... we need to circle around to the active-nodes problem in another ticket. Its a squirrely one, and I'd like to put removing the concept on that table as a potential solution, so I'd rather punt the discussion if thats okay. For now, catalogs/* doesn't restrict, but some of the other endpoints do: nodes, resources, facts, factsets.